setup.cfg

```diff
@@ -69,6 +69,7 @@ exclude =
   examples/
 
 [isort]
+profile=black
```

Comment:
this is another important line to make isort (for auto formatting imports) work nicely with black (for auto formatting everything else)